starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +136,17 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist(arrayWords, wordSearch) {
+  function searchWord(element) {
```

Comment:
I understand why you have done this, but not get used to define functions inside another functions if you don't have a very good reason for doing it.